chains/ethereum/config_test.go

```diff
@@ -12,25 +12,31 @@ import (
 	"github.com/ethereum/go-ethereum/common"
 )
 
-func TestParseChainConfig(t *testing.T) {
-	input := core.ChainConfig{
+//createChainConfig helps create a chain config
+func createChainConfig(t *testing.T, bridge string, erc20Handler string, erc721Handler string, genericHandler string) core.ChainConfig {
```

Comment:
Unfortunately this doesn't quite replicate what actually happens when we parse a config. If the addresses are not provided the elements will not be present in the map, they won't just be empty.

Reply:
Ahh okay